staker/block_validator.go

```diff
@@ -460,6 +468,7 @@ func (v *BlockValidator) validate(ctx context.Context, validationStatus *validat
 	entry := validationStatus.Entry
 	defer func() {
 		atomic.AddInt32(&v.atomicValidationsRunning, -1)
+		validatorPendingValidationsCounter.Dec(1)
```

Comment:
Prometheus Counters are typically only increasing.  Use a Gauge if you also need to decrement

Reply:
fixed, thanks